src/gui/gui_msgs.h

```diff
@@ -37,45 +37,62 @@ constexpr char help_msg[] =
 
 List of common options:
 
-  --printconf          Print the location of the default configuration file.
+  --printconf              Print the location of the default configuration file.
 
-  --editconf           Open the default configuration file in a text editor.
+  --editconf               Open the default configuration file in a text editor.
 
-  -c <command>         Run the specified DOS command before running FILE.
-                       Multiple commands can be specified.
+  --eraseconf              Delete the configuration file.
 
-  -noautoexec          Don't perform any [autoexec] actions.
+  --erasemapper            Delete the mapper file.
 
-  -noprimaryconf       Don't read settings from the primary configuration file
-                       located in your user folder.
+  --startmapper            Run the mapper GUI.
 
-  -nolocalconf         Don't read settings from "dosbox.conf" if present in
-                       the current working directory.
+  --securemode             Start in secure mode.
```

Comment:
I don't know what secure mode does exactly.  Can you help with a description here?

Reply:
From the DOSBox X manual:

> Enables DOSBox-X’s secure mode. The [config] and [autoexec] sections of the loaded configuration file will be skipped, and commands such as MOUNT and IMGMOUNT are disabled.

https://github.com/joncampbell123/dosbox-x/wiki/DOSBox%E2%80%90X%E2%80%99s-Command%E2%80%90Line-Options

You can use this with a few changes.

Reply:
Good stuff.  I removed the DOSBox-X name and used that.